execution/group_by.go

```diff
@@ -32,10 +32,13 @@ type GroupBy struct {
 	outEventTimeField octosql.VariableName
 
 	triggerPrototype TriggerPrototype
+
+	garbageCollectorBoundary int // number of minutes subtracted from watermark to create boundary for garbage collection
```

Comment:
Following up on the above - if you added such a `struct` then these comments explaining which field means what, would only need to appear once in the `struct` definition